### PR TITLE
Binary search

### PR DESCRIPTION
**Pull request by MerrybyPractice**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 #Data Structures and Algorithms, the Java Version 
 
-1. [ArrayShift](/Users/MaryCimakasky/codefellows/401/java_data_structures_and_algo/src/main/java/ArrayShift.java)
\ No newline at end of file
+1. [ArrayShift](/Users/MaryCimakasky/codefellows/401/java_data_structures_and_algo/src/main/java/ArrayShift.java)
+    ![whiteboard](/Users/MaryCimakasky/codefellows/401/java_data_structures_and_algo/assets/IMG_1614.jpeg)
```
Comment (JessLovell) [inline README.md:4]: These file paths don't work! Try using ./assets/IMG_1614.jpeg instead.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+2. [Binary Search](/Users/MaryCimakasky/codefellows/401/java_data_structures_and_algo/build/classes/java/main/BinarySearch.class)
+        ![whiteboard](/Users/MaryCimakasky/codefellows/401/java_data_structures_and_algo/assets/57274443_2248355075425569_8806547144405155840_n.jpg)
```
Comment (JessLovell) [inline README.md:6]: These file paths don't work! Try using ./assets/57274443_2248355075425569_8806547144405155840_n.jpg instead.

File: src/main/java/BinarySearch.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                returnValue = i;
+                break;
+            } else if (i == (arr.length - 1)) {
+                returnValue = -1;
+            }
+        }
+
+        return returnValue;
+    }
+}
```
Comment (JessLovell) [inline src/main/java/BinarySearch.java:29]: add new line at end of files.

File: src/test/java/BinarySearchTest.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class BinarySearchTest {
+
+    @Test
+    public void binarySearchTest() {
+        //happy path
+
```
Comment (JessLovell) [inline src/test/java/BinarySearchTest.java:12]: I know you're planning to work on this later so no worries right now.

File: README.md (hunk 1 above)
Comment (MerrybyPractice) [reply README.md:4]: Thank you! I think I just copied and pasted in IDE, but Im implementing this and we will see how it goes!

File: src/main/java/BinarySearch.java (hunk 3 above)
Comment (MerrybyPractice) [reply src/main/java/BinarySearch.java:29]: Wow, I am so used to VS yelling at me constantly about this I guess I forgot to remember?

**Pull request merged**